nixos/modules/services/networking/ircd-hybrid/builder.sh

```diff
@@ -1,3 +1,4 @@
+if [ -e .attrs.sh ]; then source .attrs.sh; fi
```

Comment:
Why don't we source this at the beginning of stdenv setup?

Reply:
$stdenv variable will not be set for the source below

The better way to fix these would be to do https://github.com/NixOS/nixpkgs/commit/983382de5b67a37b050acdd08aca8c6fde4e9f3a but I wanted to keep the diffs small

Reply:
$stdenv isn't exported?

Reply:
with structuredAttrs all things are passed with .attrs.sh/.attrs.json and nothing is in env ( i have not confirmed this yet)